README.md

```diff
@@ -34,29 +34,81 @@ See [TODO.md](https://github.com/cockroachdb/cockroach/blob/master/TODO.md)
 
 ## Running Cockroach
 
-Don't have (a recent version > 1.2 of) Docker? Follow the [instructions for installing Docker on your host system](http://docs.docker.com/installation/). If you run into trouble below, check first that you're not running an old version.
+Getting started is most convenient using a recent version (>1.2) of [Docker](http://docs.docker.com/installation/).
 
 If you don't want to use Docker,
 * set up the dev environment (see [CONTRIBUTING.md](CONTRIBUTING.md))
 * `make build`
-* replace `docker [...] [run|build] [...]` by `./cockroach` below.
+* ignore the initial calls to `docker` below.
 
 #### Bootstrap and talk to a single node
 
+Getting a Cockroach node up and running is easy. If you have the `cockroach` binary, skip over the next shell session. Most users however will want to run the following:
+
 ```bash
-$ docker run -d -p 8080:8080 "cockroachdb/cockroach" \
-    init -rpc="localhost:0" \
-    -stores="ssd=$(mktemp -d)"
+# Get the latest image from the registry. Skip if you already have an image
+# or if you built it yourself.
+$ docker pull cockroachdb/cockroach
+[...]
+# Open a shell on a Cockroach container.
+$ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
+root@82cb657cdc42:/cockroach#
 ```
-This bootstraps and starts a single node with one temporary RocksDB instance in the background (remove the `-d` flag if you want to see stdout).
-Now let's talk to this node. You can use the [REST Explorer at
-localhost:8080](http://localhost:8080) or talk directly to the API:
+
+Now we're in an environment that has everything set up, and we start by firing up a node:
+
+```bash
+$ ./cockroach init -http 0.0.0.0:8080 -stores ssd="$(mktemp -d /tmp/dbXXX)" &> node.log &
```

Comment:
It's because of this:

```
	if strings.HasPrefix(httpAddr, ":") {
		httpAddr = s.host + httpAddr
	}
```

Feels like I should be able to just take this out without losing anything. What do you think @petermattis?